pom.xml

```diff
@@ -10,14 +10,14 @@
     <properties>
         <!-- Pull the build.number from the build system and use it in the manifest of the jar -->
         <build.number>999</build.number>
-        <build.version>1.0.0</build.version>
+        <build.version>1.0.1</build.version>
```

Comment:
I don't think we need to bump the version here, as we follow a different pattern for our open source repos.  The version and tag is created after we merge to master by clicking on "Draft a new release" in the "Release" tab off the main project page.